base/multi.jl

```diff
@@ -666,7 +760,36 @@ end
 hash(r::AbstractRemoteRef, h::UInt) = hash(r.whence, hash(r.id, h))
 ==(r::AbstractRemoteRef, s::AbstractRemoteRef) = (r.whence==s.whence && r.id==s.id)
 
+"""
+    Base.remoteref_id(r::AbstractRemoteRef) -> RRID
```

Comment:
It doesn't seem to be referenced in the manual at all, or anywhere else in `stdlib`. Might be ok to keep the docstring only?